### DISCUSSION TITLE
amplifier standby logic with GPIO

### DISCUSSION THREAD

**Discussion by klaus-liebler** [MusicPlayerDaemon/MPD#1715] [Ideas]
Hi there,
I would like to use mpd running on a ROCK 5B (Raspberry Pi-like SBC) as core element of a home sound system (and very sophisticated door bell:-). To save energy, I want to use a GPIO pin to switch my amplifier into standby when no sound is played.

Is there a way to let mpd itself toggle a GPIO pin, when a sound is played? If not, maybe somebody can give me advice, how and where to implement this?

Thank you and best regards

Klaus

**Comment by Hawk777** [comment]
Could you do this, rather than inside MPD itself, by writing a client for it which sends the `status` command to check the current status, then uses `idle` to wait until the status changes, and updates the GPIO based on that? https://mpd.readthedocs.io/en/latest/protocol.html
